packages/core/src/amazonq/webview/messages/messageDispatcher.ts

```diff
@@ -29,8 +31,8 @@ export function dispatchWebViewMessagesToApps(
                  * This would be equivalent of the duration between "user clicked open q" and "ui has become available"
                  * NOTE: Amazon Q UI is only loaded ONCE. The state is saved between each hide/show of the webview.
                  */
-                telemetry.webview_load.emit({
```

Comment:
Once this is done for all the webviews, will `webview_load` be deprecated in favor of the more granular metrics?

Reply:
Yup, `webview_x` will be deprecated for something like `toolkit_moduleX`. So we'll also drop `webview_error` as well